scripts/circle-ci/run-circle-tests.sh

```diff
@@ -6,5 +6,5 @@ SHARED=':atlasdb-tests-shared:check'
 case $CIRCLE_NODE_INDEX in
     0) ./gradlew --parallel check -x $CASSANDRA -x $SHARED ;;
     1) ./gradlew --parallel $CASSANDRA ;;
-    2) ./gradlew --parallel $SHARED ;;
-esac
\ No newline at end of file
+    2) ./gradlew --parallel $SHARED && cd docs/; make html ;;
```

Comment:
We could only have it in the publish step, but then it would fail later (and notably would only fail on the build of develop, not each individual build since it is only published off of develop).  Secondly we can't just use this build because the container it's running off of isn't guaranteed to have done the build.